web_m2x_options/static/src/js/form.js

```diff
@@ -70,7 +65,7 @@ openerp.web_m2x_options = function (instance) {
             if (_.isUndefined(this.view))
                     return this._super.apply(this, arguments);
                 if (!_.isUndefined(this.view.ir_options['web_m2x_options.limit'])) {
-                this.limit = parseInt(this.view.ir_options['web_m2x_options.limit']);
+                this.limit = parseInt(this.view.ir_options['web_m2x_options.limit'],10);
```

Comment:
Oops sorry, I mistaked the diff, somehow it was in my mind that you were *removing* it, while you were adding it. Then for get my comments about this 😅